src/Dev/SapphireTest.php

```diff
@@ -1239,6 +1239,15 @@ protected function mockSleep(int $seconds): DBDatetime
      */
     private function setI18nLocale(): void
     {
+        // Skip setting the locale for excluded test classes that are known to cause a
+        // LogicException when calling getCurrentRelativePath(), and we can be sure
+        // do not require the locale to be set to en_US
+        $excluded = [
+            'SilverStripe\\BehatExtension\\Tests\\SilverStripeContextTest',
+        ];
+        if (in_array(get_class($this), $excluded)) {
```

Comment:
```suggestion
        if (in_array(static::class, $excluded)) {
```
`get_class($this)`  feels like something magic might be happening, even though these two solutions are identical in practice. I wouldn't bother suggesting this if it was the only change, but since there's another change to make as well we may as well tidy this up

Reply:
Updated